Wstack-protector

Add g++ flag `-Wstack-protector` and you get the warning:
 
```
imgui/imgui_draw.cpp: In member function ‘void ImDrawList::AddPolyline(const ImVec2*, int, ImU32, bool, float)’:
imgui/imgui_draw.cpp:669:6: warning: stack protector not protecting local variables: variable length buffer [-Wstack-protector]
 void ImDrawList::AddPolyline(const ImVec2* points, const int points_count, ImU32 col, bool closed, float thickness)

imgui/imgui_draw.cpp: In member function ‘void ImDrawList::AddConvexPolyFilled(const ImVec2*, int, ImU32)’:
imgui/imgui_draw.cpp:857:6: warning: stack protector not protecting local variables: variable length buffer [-Wstack-protector]
 void ImDrawList::AddConvexPolyFilled(const ImVec2* points, const int points_count, ImU32 col)
```

--- a/imgui_draw.cpp
+++ b/imgui_draw.cpp
@@ -71,6 +71,7 @@ Index of this file:
 #pragma GCC diagnostic ignored "-Wunused-function"          // warning: 'xxxx' defined but not used
 #pragma GCC diagnostic ignored "-Wdouble-promotion"         // warning: implicit conversion from 'float' to 'double' when passing argument to function
 #pragma GCC diagnostic ignored "-Wconversion"               // warning: conversion to 'xxxx' from 'xxxx' may alter its value
+#pragma GCC diagnostic ignored "-Wstack-protector"          // warning: stack protector not protecting local variables: variable length buffer
 #if __GNUC__ >= 8
 #pragma GCC diagnostic ignored "-Wclass-memaccess"          // warning: 'memset/memcpy' clearing/writing an object of type 'xxxx' with no trivial copy-assignment; use assignment or value-initialization instead
 #endif

